### PR TITLE
feat(tokens): add new blue-75 token, update aliases per spec

### PR DESCRIPTION
**Pull request by williamernest**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-1111

## What is the new behavior?
Adds a new blue-75 token, remaps a disabled and active token per the spec. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/core/build/tokens.ts [hunk 1]
```diff
@@ -82,6 +82,7 @@ const color = {
   },
   blue: {
     50: token([198, 100, 95]),
+    75: token([198, 100, 94]),
```
Comment (kevinbuhmann) [inline projects/core/build/tokens.ts:85]: We need an updated design for https://v16-beta.clarity.design/documentation/color. Adding blue-75 will throw off the other columns.

Comment (williamernest) [reply projects/core/build/tokens.ts:85]: @kevinbuhmann Good catch. Can you file a ticket in JIRA to track that and assign it to the appropriate person?

**Comment by kevinbuhmann** [comment]
I put the reviewer back to Patrick because I am not knowledgeable of these changes.

**Review by kevinbuhmann — APPROVED** [review]
I am not knowledgeable of whether or not these changes are correct, but the code seems fine with what I know.

File: projects/core/build/tokens.ts (hunk 1 above)
Comment (kevinbuhmann) [reply projects/core/build/tokens.ts:85]: CDE-1187

Comment (Partick) [reply projects/core/build/tokens.ts:85]: This is correct.

**Comment by Partick** [comment]
Approved! Good to go!

**Review by Partick — APPROVED** [review]
Color is correct and I can see it updated for the active background. Good to go!

**Pull request merged**